CHANGES.rst

```diff
@@ -1,3 +1,447 @@
+
+astropy.cosmology
+^^^^^^^^^^^^^^^^^
+
+- Cosmology instance can be parsed from or converted to a YAML string using
+  the new "yaml" format in Cosmology's ``to/from_format`` I/O. [#12279]
+
+- Register "astropy.row" into Cosmology's to/from format I/O, allowing a
+  Cosmology instance to be parse from or converted to an Astropy Table Row. [#12313]
+
+- A method ``clone`` has been added to ``Parameter`` to quickly deep copy the
+  object and change any constructor argument.
+  A supporting equality method is added, and ``repr`` is enhanced to be able to
+  roundtrip -- ``eval(repr(Parameter()))`` -- if the Parameter's arguments can
+  similarly roundtrip.
+  Parameter's arguments are made keyword-only. [#12479]
+
+- Add methods ``Otot`` and ``Otot0`` to FLRW cosmologies to calculate the total
+  energy density of the Universe. [#12590]
+
+- Add property ``is_flat`` to cosmologies to calculate the curvature of the Universe.
+
+  ``Cosmology`` is now an abstract class and subclasses must override the
+  abstract property ``is_flat``. [#12606]
+
+- For converting a cosmology to a mapping, two new boolean keyword arguments are
+  added: ``cosmology_as_str`` for turning the class reference to a string,
+  instead of the class object itself, and ``move_from_meta`` to merge the
+  metadata with the rest of the returned mapping instead of adding it as a
+  nested dictionary. [#12710]
+
+- Register format "astropy.cosmology" with Cosmology I/O. [#12736]
+
+- Cosmological equivalency (``Cosmology.is_equivalent``) can now be extended
+  to any Python object that can be converted to a Cosmology, using the new
+  keyword argument ``format``.
+  This allows e.g. a properly formatted Table to be equivalent to a Cosmology. [#12740]
+
+- The new module ``cosmology/tests/helper.py`` has been added to provide tools
+  for testing the cosmology module and related extensions. [#12966]
+
+- A new property ``nonflat`` has been added to flat cosmologies
+  (``FlatCosmologyMixin`` subclasses) to get an equivalent cosmology, but of the
+  corresponding non-flat class. [#13076]
+
+- ``clone`` has been enhanced to allow for flat cosmologies to clone on the
+  equivalent non-flat cosmology. [#13099]
+
+- ``cosmology`` file I/O uses the Unified Table I/O interface, which has added
+  support for reading and writing file paths of the form ``~/file.ecsv`` or
+  ``~<username>/file.ecsv``, referring to the home directory of the current user
+  or the specified user, respectively. [#13129]
+
+astropy.io.ascii
+^^^^^^^^^^^^^^^^
+
+- Simplify the way that the ``converters`` argument of ``io.ascii.read`` is
+  provided. Previously this required wrapping each data type as the tuple returned
+  by the ``io.ascii.convert_numpy()`` function and ensuring that the value is a
+  ``list``. With this update you can write ``converters={'col1': bool}`` to force
+  conversion as a ``bool`` instead of the previous syntax ``converters={'col1':
```

Comment:
Is Sphinx smart enough to allow the double backtick with endline in between?

Reply:
Yes it works: https://astropy--13270.org.readthedocs.build/en/13270/changelog.html